app/code/Magento/Developer/Model/Logger/Handler/Debug.php

```diff
@@ -3,18 +3,23 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```

Comment:
If we add this declaration I would recommend to specify types for methods parameters
This may be a more backward compatible solution
See an example for [laminas/laminas-escaper library](https://github.com/laminas/laminas-escaper/releases/tag/2.8.0)